tests/server/region_syncer/region_syncer_test.go

```diff
@@ -151,7 +151,7 @@ func TestRegionSyncer(t *testing.T) {
 		re.Equal(region.GetLeader(), r.GetLeader())
 		re.Equal(region.GetBuckets(), r.GetBuckets())
 	}
-	re.NoError(failpoint.Disable("github.com/tikv/pd/server/syncer/noFastExitSync"))
```

Comment:
This issue may not be related to this PR change. The old path `github.com/tikv/pd/server/syncer/noFastExitSync` seems to be wrong? Should be `regionsyncer` other than `syncer`. So does this failpoint work?

Reply:
The package name is `syncer` I think.

Reply:
Get